src/core/thread/mle_router.hpp

```diff
@@ -560,6 +560,15 @@ class MleRouter : public Mle
      *
      */
     void SetCcmEnabled(bool aEnabled) { mCcmEnabled = aEnabled; }
+
+    /**
+     * This method sets flag informing that thread protocol version check VR should be skipped
+     * when deciding if device should operate as router.
+     *
+     * @param[in]  aEnabled  TRUE if the VR check should be skipped, FALSE otherwise.
+     *
+     */
```

Comment:
```suggestion
    /**
     * This function sets whether the Security Policy TLV version-threshold for routing (VR field) is enabled.
     *
     * @param[in]  aEnabled  TRUE to enable Security Policy TLV version-threshold for routing, FALSE otherwise.
     *
     */
```